questions/307/meta_data.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": 307,
-    "result": "OK",
-    "answer": "0",
+    "result": "CE",
+    "answer": "",
     "state": "PUB",
     "difficulty": 2
-}
\ No newline at end of file
+}
```

Comment:
nit: usually, there's no EOL in `meta_data.json`

Reply:
This doesn't matter, these files are just read by a script and die with this repo.